Libraries/Opc.Ua.Client/Session/Session.cs

```diff
@@ -4358,6 +4358,15 @@ private Node ProcessReadResponse(
                             continue;
                         }
 
+                        // ignore errors with attributes, which were not in 1.03 
+                        if (values[ii].StatusCode == StatusCodes.BadNodeIdUnknown)
+                        {
+                            if (attributeId == Attributes.AccessLevelEx)
```

Comment:
why not add it to the table of optional attributes below? It has apparently being missed at a certain point when the project was updated to 1.04

Reply:
I miss also as optional from the table: ArrayDimesnions, InverseName, MinimumSamplingInterval, could you fix them as well?